packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -65,21 +67,21 @@ const styles: ComponentStyles<CallToActionClassNameContract, DesignSystem> = (
     return {
         callToAction: {
             fontSize: "15px",
-            display: "inline-block",
+            display: "inline-flex",
             maxWidth: "100%",
             border: "2px solid transparent",
-            padding: localizeSpacing(direction)("13px 22px 11px 24px"),
-            [applyLocalizedProperty("marginLeft", "marginRight", direction)]: "-10px",
+            padding: "0 12px",
             lineHeight: "1",
             textDecoration: "none",
             whiteSpace: "nowrap",
-            verticalAlign: "bottom",
-            marginTop: "16px",
             transition: "all 0.2s ease-in-out",
             "&:hover, &:focus": {
                 outline: "none",
                 "& $callToAction_glyph": {
-                    left: "8px",
+                    transform:
+                        direction === Direction.ltr
+                            ? "translateX(4px)"
+                            : "rotate(180deg) translateX(4px)",
```

Comment:
should translate be `-4px`?

Reply:
@nicholasrice can you pull down? it seems odd, but this was actually working as expected...

Reply:
yep, thats... not what I expected. Looks good!